src/Symfony/Component/Form/Extension/Core/Type/DateType.php

```diff
@@ -320,6 +321,9 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedTypes('months', 'array');
         $resolver->setAllowedTypes('days', 'array');
         $resolver->setAllowedTypes('input_format', 'string');
+        $resolver->setAllowedTypes('calendar', ['null', \IntlCalendar::class]);
+
+        $resolver->setInfo('calendar', 'The calendar to use for formatting and parsing the date. The value should be one of the \IntlDateFormatter calendar constants or an instance of the \IntlCalendar to use.');
```

Comment:
This information looks wrong. The allowed types don't allow the integer constants

Reply:
I'm having a look. Thanks for raising this issue!